pkgs/development/python-modules/mesonpep517/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, meson
+, ninja
+, intreehooks
+, pytoml
+, pythonOlder
+}:
+
+# TODO: offer meson as a Python package so we have dist-info folder.
```

Comment:
It's silly, actually. We need it is as a Python package if packages start adding it as a build system requirement. However, it should not be needed considering meson is only an *application*; they do not actually import it.